ledger/light/pulsemanager/pulsemanager.go

```diff
@@ -410,9 +411,11 @@ func (m *PulseManager) Set(ctx context.Context, newPulse insolar.Pulse, persist
 		return nil
 	}
 
-	// Run only on material executor.
-	// execute only on material executor
-	if m.NodeNet.GetOrigin().Role() == insolar.StaticRoleLightMaterial && oldPulse != nil && prevPN != nil {
+	if oldPulse != nil && prevPN != nil {
+		err = m.WriteManager.CloseAndWait(ctx, oldPulse.PulseNumber)
+		if err != nil {
```

Comment:
We should log the error and move on. We can't stop all the processing because of this.

Reply:
done